woocommerce/woocommerce
# Standardize linting: e2e js packages

### All Submissions:






### Changes proposed in this Pull Request:

In an effort to standardize lint scripts across the monorepo, this PR makes sure the e2e-* js packages are equiped with linting scripts `lint` and `lint:fix` and work in accordance with the rules specified by `@woocommerce/eslint-plugin`.

This PR also runs the `lint:fix` script to fix any errors found. Note, this did not solve all of them, but fixing the many hundreds by hand is out of scope of this PR.

Patially addresses some points found in https://github.com/woocommerce/woocommerce/issues/32653

### How to test the changes in this Pull Request:

1. `pnpm install`
2. In an e2e js package, `pnpm run lint`
3. See some errors appear
4. Introduce a whitespace error and `pnpm run lint:fix`. See the errors appear and the one you introduced fixed.

### Other information:




### FOR PR REVIEWER ONLY:
